arch/arm/boot/dts/overlays/vc4-kms-dsi-waveshare-panel-overlay.dts

```diff
@@ -120,6 +126,8 @@
 				   <&touch>, "touchscreen-inverted-x?",
 				   <&touch>, "touchscreen-inverted-y?";
 		8_8_inch = <&panel>, "compatible=waveshare,8.8inch-panel";
+		13_3_60fps_inch = <&panel>, "compatible=waveshare,13.3inch-60fps-panel";
+		13_3_30fps_inch = <&panel>, "compatible=waveshare,13.3inch-30fps-panel";
```

Comment:
These names look wrong - `13_3_inch_60fps` and `13_3_inch_30fps` make more sense.

Reply:
Yeah, I found the problem. I'm working on it.
I think it would be more appropriate to read `13_3_inch_4lane`.